core/views.py

```diff
@@ -1,72 +1,57 @@
-    """ Initialize the data for the app"""
-    all_metrics = load_summary_metric('all')
-    all_metrics["Facility Name"] = standardize_cms_name(all_metrics["Facility Name"])
-    all_metrics["Address"] = all_metrics["Address"].str.lower()
-    return all_metrics
-
+# Initialize the data for the app
+gmaps = GoogleMapsClient(key='AIzaSyD2Rq696ITlGYFmB7mny9EhH2Z86Xekw4o')
 summary_metrics = load_summary_metrics()
 
 @timeit
 def index(request, path=None):
+    """ The main landing route for the app"""
+    # Parse request params to get google query args
     search_string = request.GET.get("search")
     location_string = request.GET.get("location")
-    radius_string = request.GET.get("radius")
-    gmaps_places_args = {}
+    radius = request.GET.get("radius") # in meters
+
+    # Query google maps for places
+    places_data = {}
     if search_string:
+        # Google maps query args
+        # Example query: curl -L -X GET 'https://maps.googleapis.com/maps/api/place/textsearch/json?query=t&location=42.3675294%2C-71.186966&radius=10000&key=AIzaSyD2Rq696ITlGYFmB7mny9EhH2Z86Xekw4o'
+        gmaps_places_args = {}
         gmaps_places_args["query"] = search_string.strip()
-    if location_string:
-        split_location_string = location_string.strip().split(",")
-        gmaps_places_args["location"] = [float(split_location_string[1]),float(split_location_string[0])]
-    #curl -L -X GET 'https://maps.googleapis.com/maps/api/place/textsearch/json?query=t&location=42.3675294%2C-71.186966&radius=10000&key=AIzaSyD2Rq696ITlGYFmB7mny9EhH2Z86Xekw4o'
+        if location_string:
+            split_location_string = location_string.strip().split(",")
+            gmaps_places_args["location"] = [float(split_location_string[1]),float(split_location_string[0])]
 
-    gmaps = googlemaps.Client(key='AIzaSyD2Rq696ITlGYFmB7mny9EhH2Z86Xekw4o')
-    places_results = []
-    if search_string:
         print("search args", gmaps_places_args)
-        places_results = gmaps.places(**gmaps_places_args, radius=radius_string or 10000)
-        for place_result in places_results['results']:
-            place_detail = gmaps.place(place_id=place_result["reference"])
-            place_detail = place_detail["result"]
-            if "formatted_phone_number" in place_detail:
-                place_result["phone_number"] = place_detail["formatted_phone_number"]
-            place_result = add_metrics_to_place(summary_metrics, place_result)
+        places_data = get_places(gmaps, gmaps_places_args, radius= int(float(radius)) if radius else 10000)
+        valid_results = filter_place_results(places_data["results"]) # Query args only support 1 type filter, so we filter results aftwards
+        update_place_results(valid_results, gmaps, summary_metrics) # Updates in place
+        places_data['results'] = valid_results
+
+    # Context for the front end
     context = {
-        'google_places_data' : places_results,
+        'google_places_data' : places_data,
```

Comment:
"data" is super generic but "results" implied that it came from a search @ @

Reply:
I agree, I don't like places_data as a variable name, but it matches placesData on the frontend, which is why I chose it. And the only reason I renamed it in the first place is place_results["results"] was a confusing when saying places_results['results'] = valid_results